src/azure-cli/azure/cli/command_modules/botservice/_exception_handler.py

```diff
@@ -7,13 +7,13 @@
 
 
 def bot_exception_handler(ex):
-    from azure.mgmt.botservice.models import ErrorException
+    from azure.core.exceptions import HttpResponseError
     from msrestazure.azure_exceptions import CloudError
     from msrest.exceptions import ClientRequestError  # pylint: disable=import-error
-    if isinstance(ex, ErrorException):
+    if isinstance(ex, HttpResponseError):
         message = 'An error occurred. {0}: {1}'.format(
-            ex.error.error.code,
-            ex.error.error.message
+            ex.error.code,
+            ex.error.message
         )
         raise CLIError(message)
```

Comment:
CLIError is dropped. Please use error types defined in this [guideline](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md#available-error-types)

Reply:
fixed